tg_fm_news.py

```diff
@@ -5,15 +5,25 @@
 import time
 import requests
 import telebot
+import psycopg2
+import datetime
 from bs4 import BeautifulSoup
 from tg import BOT_TOKEN, chat_id
 
 should_exit = False  # Флаг для завершения
 bot = telebot.TeleBot(BOT_TOKEN)
 
+try:
+    #Connecting to the database
+    conn = psycopg2.connect(dbname='NewsDatabase', user='postgres', password='admin', host='localhost')
+except:
+    # if the connection fails, a message will be displayed in STDOUT
+    print('Can`t establish connection to database')
```

Comment:
Terminate application with non-zero exit code in case pf connection failure.

Reply:
fixed

Reply:
Fixed

Reply:
With NON-zero exit code.